CppCoreGuidelines.md

```diff
@@ -7,7 +7,7 @@ Editors:
 * [Bjarne Stroustrup](http://www.stroustrup.com)
 * [Herb Sutter](http://herbsutter.com/)
 
-This document is a very early draft. It is inkorrekt, incompleat, and pÂµÃ¸oorly formatted.
+This document is a very early draft. It is inkorrekt, incompleat, and pÂµÃoorly formatted.
```

Comment:
yes, that char also occured in other places, I think it's a twisted comma, so I check for it in the makefile, so it had to go. The others still make the point (and besides, I think the sentence is not as valid as it once was, except for incompleteness).